src/generator/data/README

```diff
@@ -0,0 +1,3 @@
+The files located in this directory is part of Unicode Character
+Database.  Their use, reproduction, distribution are permitted under the
+term appeared in UnicodeData.html.
```

Comment:
Grammar (and content) is a bit buggy here. This should say "The files located in this directory are taken verbatim from the Unicode Character Database. Their use, reproduction, and distribution are permitted under the terms in UnicodeData.html"